.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,27 @@
+name: PR
+
+on:
+    pull_request:
+      types: [opened, reopened, synchronize]
+
+jobs:
+    soundness:
+        name: Soundness
+        uses: apple/swift-nio/.github/workflows/soundness.yml@main
```

Comment:
Is it on purpose that we reference files from Swift NIO project ?

Reply:
yes!